<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ExtendedTabWidget" name="main_tab">
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_open_recent">
     <property name="title">
      <string>Open &amp;Recent</string>
     </property>
     <property name="icon">
      <iconset theme="document-open-recent">
       <normaloff/>
      </iconset>
     </property>
     <addaction name="action_no_recent_files"/>
    </widget>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="menu_open_recent"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="action_save_all"/>
    <addaction name="action_reload"/>
    <addaction name="separator"/>
    <addaction name="action_print"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
    <addaction name="action_close_all"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_image">
    <property name="title">
     <string>&amp;Image</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_animation">
    <property name="title">
     <string>A&amp;nimation</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_about_qt"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menu_docks">
     <property name="title">
      <string>&amp;Docks</string>
     </property>
    </widget>
    <widget class="QMenu" name="menu_toolbars">
     <property name="title">
      <string>Tool&amp;bars</string>
     </property>
    </widget>
    <addaction name="menu_docks"/>
    <addaction name="menu_toolbars"/>
   </widget>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <widget class="QMenu" name="menu_plugins">
     <property name="title">
      <string>Pl&amp;ugins</string>
     </property>
     <property name="icon">
      <iconset theme="preferences-plugin">
       <normaloff/>
      </iconset>
     </property>
    </widget>
    <addaction name="menu_plugins"/>
    <addaction name="action_preferences"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_image"/>
   <addaction name="menu_animation"/>
   <addaction name="menu_tools"/>
   <addaction name="menu_view"/>
   <addaction name="menu_settings"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_new">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action_save_all">
   <property name="icon">
    <iconset theme="document-save-all"/>
   </property>
   <property name="text">
    <string>Save A&amp;ll</string>
   </property>
  </action>
  <action name="action_print">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="document-print"/>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset theme="document-close"/>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_no_recent_files">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;No Recent Files</string>
   </property>
  </action>
  <action name="action_close_all">
   <property name="icon">
    <iconset theme="window-close"/>
   </property>
   <property name="text">
    <string>Clos&amp;e All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="action_preferences">
   <property name="icon">
    <iconset theme="configure"/>
   </property>
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="action_reload">
   <property name="icon">
    <iconset theme="document-revert"/>
   </property>
   <property name="text">
    <string>Reloa&amp;d</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtendedTabWidget</class>
   <extends>QTabWidget</extends>
   <header>extended_tab_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_new</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>documentNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_open</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>documentOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_save</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>documentSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_save_as</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>documentSaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_close</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>documentClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_close_all</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>documentCloseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_reload</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>documentReload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>documentNew()</slot>
  <slot>documentOpen()</slot>
  <slot>documentSave()</slot>
  <slot>documentSaveAs()</slot>
  <slot>updateTitle()</slot>
  <slot>documentClose()</slot>
  <slot>documentCloseAll()</slot>
  <slot>documentReload()</slot>
 </slots>
</ui>
